### ISSUE TITLE
JSX: function component is not beautified well

### ISSUE THREAD

**Issue by dai-shi** [dai-shi/es-beautifier#7]
Given the code:

```
const App = () => <div><span>aaaaaaaaaaaaaaaa</span><span>bbbbbbbbbbbbbbbb</span></div>;
```
The expected result:

```
const App = () => (
  <div>
    <span>aaaaaaaaaaaaaaaa</span>
    <span>bbbbbbbbbbbbbbbb</span>
  </div>
);
```

**Comment by dai-shi** [comment]
It looks like the jsx-wrap-multiline rule doesn't put parentheses.

The relevant issue: https://github.com/yannickcr/eslint-plugin-react/issues/790

**Comment by dai-shi** [comment]
fixed in v1.0.0.
